connector/common/__init__.py

```
@@ -0,0 +1,4 @@
+from .enums import HTTPMethod
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)